download.html

```diff
@@ -24,6 +24,127 @@ <h1>Downloads</h1>
    <div class="row">
       <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 cDownloadsHeader">
          <div class="cFeaturedVersion">
+
```

Comment:
We should change the version Drop downs in the entire website right?

Reply:
That will be done later.